include/utils.h

```diff
@@ -8,7 +8,7 @@
  * All utility fonctions are referenced and implemented inside this file.
  */
 
-#include "../extsrc/pugixml/src/pugixml.hpp"
+#include "pugixml.hpp"
```

Comment:
why "" here and <> in DataGenerator ?

Reply:
Includes are inconsistent in this project; I'll fix them as I go over each file.

It boils down to:
- it works as it is now because of the lookup rules for include files
- for consistency, we should change `""` to `<>` for system includes and keep `""` for local includes
- we should do that only after we make a decision about the current `src`, `include` and `test` directories